src/collections/_documentation/development/sdk-dev/event-payloads/index.md

```diff
@@ -54,22 +54,21 @@ The following attributes are required for all events.
 
 `timestamp`
 
-: Indicates when the logging record was created (in the Sentry SDK). The Sentry
-  server assumes the time is in UTC. The timestamp should be in ISO 8601 format,
-  without a timezone.
+: Indicates when the logging record was created (in the Sentry SDK). The format
+  is either a string as defined in [RFC
+  3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float)
+  value representing the [number of seconds that have elapsed since the Unix
+  epoch](https://en.wikipedia.org/wiki/Unix_time).
```

Comment:
The format description is the same as for `Breadcrumb.timestamp`.

I intentionally did not change other timestamps that are not part of ingestion, e.g. multiple parts of the Web API, because I did not research what is accepted in each case.